packages/bot-web-ui/src/stores/app-store.js

```diff
@@ -141,7 +141,12 @@ export default class AppStore {
         this.disposeLandingCompanyChangeReaction = reaction(
             () => client.landing_company_shortcode,
             () => {
-                if (client.has_maltainvest_account || client.is_options_blocked) {
+                if (
+                    (!client.is_logged_in && client.is_eu_country) ||
+                    client.has_maltainvest_account ||
+                    isEuResidenceWithOnlyVRTC(client.residence, client.active_accounts) ||
```

Comment:
When we are checking only VRTC it is not imporant that what the active account is because it is only one.

Reply:
There were 2 options to pass: `client.accounts` and `client.active_accounts`
No difference in what we want to do here regarding them.